recipes-core/bluebinder/bluebinder.bb

```diff
@@ -6,7 +6,7 @@ SECTION = "webos/support"
 LIC_FILES_CHKSUM = "file://bluebinder.c;beginline=1;endline=27;md5=430727b8efeca344ab89eeb635b4fa79"
```

Comment:
This file shouldn't be needed as it's available in `meta-asteroid`.
(I get that this is a commit to fix the syntax changed with the new Honster Yocto build, just letting you know that it's not needed here :smile:  )

Reply:
Recipe Removed.